docs/devops-guidelines/DevOps-Guidelines.md

```diff
@@ -0,0 +1,20 @@
+# DevOps Approach
+
+"DevOps is cultural fusion of People, Process, and Technology to accelerate deliery of Business Value"
```

Comment:
"DevOps is cultural fusion of People, Process, and Technology to accelerate delivery of Business Value"